### ISSUE TITLE
Can the popups be customised?

### ISSUE THREAD

**Issue by DhanushNehru** [DhanushNehru/breakout-game#3]
Currently after the game is played, either lost or won the popup is just a normal prompt

Can this be customised with html or css & make it to look better

**Comment by ASANIYAN** [comment]
Hello, I would love to work on customizing the popup to make it look better.

**Comment by ASANIYAN** [comment]
Yes, it can. Do you have any pop up designs in mind or I can go for it

**Comment by DhanushNehru** [comment]
You can go with your preference

**Comment by Akash2866** [comment]
Can I work on it under hacktoberfest23, please??

**Comment by DhanushNehru** [comment]
Sure @Akash2866
